code/presentation/mb-client/app.config.ts

```diff
@@ -33,6 +33,11 @@ export default ({ config }: ConfigContext): ExpoConfig => ({
       backgroundColor: '#2E3C4B',
     },
     package: Env.PACKAGE,
+    config: {
+      googleMaps: {
+        apiKey: 'AIzaSyDrs_gpMZ97ldQhxHBcd28UAQcUBylz9AM',
```

Comment:
apikey shouldn't be commit, please load it through env instead

Reply:
Done